pom.xml

```diff
@@ -213,7 +213,7 @@
 			<plugin>
 				<groupId>org.codehaus.mojo</groupId>
 				<artifactId>gwt-maven-plugin</artifactId>
-				<version>1.2</version>
+				<version>2.7.0</version>
```

Comment:
We should probably move this to a property variable, so that dependent classes can override it if necessary.